src/components/Form/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Button = ({
+  icon = "",
```

Comment:
What happens if there is no icon passed like we have in the default? Does the `<FontAwesomeIcon/>` component just not render? Or do we need to add our logic to prevent an exception error?

Reply:
When I tested passing no icon, nothing renders so that's why I opted for no default on this parameter.